packages/manager/apps/dedicated/client/app/components/dedicated-cloud/operation/executionDateEdit/dedicatedCloud-operation-executionDateEdit.controller.js

```diff
@@ -40,13 +40,16 @@ export default class {
           this.$translate.instant('dedicatedCloud_OPERATIONS_success'),
         ),
       )
-      .catch((error) =>
-        this.goBack(
-          this.$translate.instant('dedicatedCloud_OPERATIONS_error', {
-            message: get(error, 'message'),
-          }),
+      .catch((error) => {
+        console.log('error', error);
```

Comment:
- console.log

Reply:
Sorry... It is fixed. Thanks for pointing this out :)